libs/core/src/components/pds-table/pds-table-head-cell/test/pds-table-head-cell.e2e.ts

```diff
@@ -0,0 +1,45 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('pds-table-head-cell', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<pds-table-head-cell></pds-table-head-cell>');
+
+    const element = await page.find('pds-table-head-cell');
+    expect(element).toHaveClass('hydrated');
+  });
+
+  it('renders with fixed styles when part attribute is set', async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+      <pds-table fixed-column>
+        <pds-table-head>
```

Comment:
no use of `pds-table-body`. Do we want to be consistent or look into making sure the schema requires the use `pds-table-body`?

Reply:
This is a test for the table head, not sure why I need to include the body?

Reply:
same comment as before. Not realizing it was jut for the `table-head` only.